### ISSUE TITLE
Adding artist, copyright and license to photos

### ISSUE THREAD

**Issue by spixi** [diaspora/diaspora#7626]
In the current implementation the Exif data of images is stripped, unless the user opts-out for this privacy setting.

However, the Exif data also may contain information about the artist, the copyright and the license. To enhance the user experience and to make it easier to deal with photos there should be the possibility to add this information to uploaded photos .

In this issue, I only refer to photos uploaded to diaspora*, not linked ones) A nice-to-have feature would include auto-suggestion via Exif tag detection.

The artist field should be mandatory, if a license is selected which requires attribution (like CC-by or LAL).

**Comment by Jeeppler** [comment]
I think it is important to attribute the artist. However, exif metadata is visible for the computer and not necessarily for the human. I would be happy to have a simple way, (extra field) or so, to attribute the artist and license of the picture. It would be nice to have those information in the figure caption.

**Comment by spixi** [comment]
Yes, this would require a dropdown box with licenses and an optional author field. The author field is required for some licenses like „All rights reserved“,  „CC-by-4.0“, „LAL-1.3“, „FDL-1.3“, „DSL“ etc.
